### ISSUE TITLE
Have Alloy left instead of middle

### ISSUE THREAD

**Issue by joaovitorborges** [PixelRick/HZD_PC_Gamer_Camera#5]
I know this was made a while ago, but was wondering if its possible to keep allow always to the left, similar to how it is when you are aiming.

**Comment by WulfeBlood** [comment]
^ I request this also, just to the right instead.

**Comment by just-thefacts** [comment]
Yes, an over the right shoulder view would be nice. As long as it does not jump back and forth like the default.
